src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
The class `FruitTransaction` seems to be missing access modifiers for its fields. According to the encapsulation principle, it's recommended to use private access modifiers for class fields and provide public getter methods to access these fields if necessary.